### PR TITLE
Implement alternating colors on frontend

### PR DESCRIPTION
**Pull request by jpoist97**

### CODE REVIEW

File: components/CreateWorkout/CreateWorkout.jsx [hunk 1]
```diff
@@ -123,30 +123,30 @@ export default ({ navigation }) => {
...
+            muscleGroups += ' ';
             muscleGroups += itemState[1].muscleGroups;
           }
           const newWorkout = {
             name,
             lastPerformed: 'n/a',
             id: nextWorkoutId,
-            muscleGroups: muscleGroups,
+            muscleGroups,
             color: itemState[0].color,
```
Comment (ctperry0301) [inline components/CreateWorkout/CreateWorkout.jsx]: If kept, the color would be stored in the database which defeats the purpose of rendering it on the front end. (Is this needed for front end rendering or redux or something?) If not, then for this task, I think you should also get rid of all instances relating color to redux and db.

**Review by ctperry0301 — CHANGES_REQUESTED** [review]
The front end rendering part works great. However, I think there is still some code lingering from when we thought we were gonna store colors in the database. Clean out that code and I think it should be good to go.

File: components/CreateWorkout/CreateWorkout.jsx (hunk 1 above)
Comment (jpoist97) [reply components/CreateWorkout/CreateWorkout.jsx]: I removed those instances when I fixed the cycle bug, they should all be gone now

**Review by ctperry0301 — APPROVED** [review]
Works Great, good stuff!

**Pull request merged**